WooCommerce/Classes/POS/Presentation/Card Present Payments/Reader Messages/PointOfSaleCardPresentPaymentCaptureErrorMessageViewModel.swift

```diff
@@ -26,7 +26,12 @@ final class PointOfSaleCardPresentPaymentCaptureErrorMessageViewModel: Observabl
     }
 
     static func == (lhs: PointOfSaleCardPresentPaymentCaptureErrorMessageViewModel, rhs: PointOfSaleCardPresentPaymentCaptureErrorMessageViewModel) -> Bool {
-        lhs.id == rhs.id
+        lhs.id == rhs.id &&
```

Comment:
Addressed a comment from the previous PR https://github.com/woocommerce/woocommerce-ios/pull/13741#discussion_r1732411976